### DISCUSSION TITLE
自定义主题，Icarus cannot load plugin number

### DISCUSSION THREAD

**Discussion by Nofated095** [ppoffice/hexo-theme-icarus#1184] [Q&A] [answered]
我 Fork 了 icarus 的仓库并修改了一部分内容后在 npm publish 发布了 hexo-theme-amane。
但在实际渲染上，主题只渲染了文章部分，其他 widget，自定义页面全都没有渲染。并在 Console 报错：Icarus cannot load plugin number。number 为从 0 -18 的循环。

不太清楚如何解决这部分问题，我在这个自定义主题中只是添加了一些  js。

https://github.com/Nofated095/blog/actions/runs/4135828747/jobs/7148829035

Nofated095/hexo-theme-amane

**Comment by ppoffice** [comment, marked as answer]
`hexo-theme-amane` 有依赖 `hexo-component-inferno`么？

**Reply by Nofated095** [reply]
应该是有的，因为 dependencies 和 icarus 的 package.json 相同，我没有修改

**Reply by ppoffice** [reply]
把这一行去掉试试：https://github.com/Nofated095/blog/blob/master/_config.yml#L101

**Reply by Nofated095** [reply]
> 把这一行去掉试试：https://github.com/Nofated095/blog/blob/master/_config.yml#L101

![0327338C-D632-4C9B-9CF6-E6EAC79EA1FC](https://user-images.githubusercontent.com/49985975/218245505-ed212ce6-bfbd-4d26-b3cc-0217b2ddbce6.png)

去掉之后 cannot load plugin 问题解决了，但是无法生成除了 posts 以外的文件，且似乎没有正确读取 _config.icarus.yml 文件，原配置好的 waline，widget，navbar 都没有加载，provider 使用的是 icarus 默认的几个 cdn。

**Reply by ppoffice** [reply]
你主题都改名字了，应该叫`_config.amane.yml`
